components/RemoveIcon.js

```diff
@@ -3,23 +3,34 @@ import PropTypes from 'prop-types';
 import IconButton from '@material-ui/core/IconButton';
 import DeleteIcon from '@material-ui/icons/Delete';
 
-function RemoveIcon(props) {
+const RemoveIcon = props => {
```

Comment:
Again, destructuring in the argument is cleaner

```suggestion
const RemoveIcon ({ propName }) => (
```